### ISSUE TITLE
Unsatisfying results and some questions

### ISSUE THREAD

**Issue by St1ckyfinger** [KunZhou9646/controllable_evc_code#4]
Hi @KunZhou9646, 

Thanks for your excellent work. I have run your scripts with the help of [this project](https://github.com/JeremyCCHsu/vae-npvc/tree/vawgan). But unfortunately, emotion is not converted anyway. I am waiting for your official training guideline.

By the way, I have two questions about the methods:

- Firstly, I find the loss function of the discriminator is inconsistent between the paper and code. Could you please give more details?

- Secondly, I am curious about the usage of F0. Why use the original f0 during training while use converted f0 in run-time conversion? 

Thanks a lot and looking forward to your reply!

**Comment by zy511361103** [comment]
hello.could you tell me how to get these features.i will be appreciate.
![image](https://user-images.githubusercontent.com/82103943/177099743-7bc1c0d2-dc1d-4e27-9ad7-6f8de6141a91.png)

**Comment by KunZhou9646** [comment]
> hello.could you tell me how to get these features.i will be appreciate. ![image](https://user-images.githubusercontent.com/82103943/177099743-7bc1c0d2-dc1d-4e27-9ad7-6f8de6141a91.png)

Hey these features are deep features obtained from the last projection layer of a pre-trained SER model.

**Comment by zy511361103** [comment]
> 

Thanks.Hope to read the user guide earlier.

**Comment by dhilu164** [comment]
> Hi @KunZhou9646,
> 
> Thanks for your excellent work. I have run your scripts with the help of [this project](https://github.com/JeremyCCHsu/vae-npvc/tree/vawgan). But unfortunately, emotion is not converted anyway. I am waiting for your official training guideline.
> 
> By the way, I have two questions about the methods:
> 
> * Firstly, I find the loss function of the discriminator is inconsistent between the paper and code. Could you please give more details?
> * Secondly, I am curious about the usage of F0. Why use the original f0 during training while use converted f0 in run-time conversion?
> 
> Thanks a lot and looking forward to your reply!

Can you tell me how you add dataset path in analyzer.py file by the way i am using ESD dataset

**Comment by dhilu164** [comment]
> > 
> 
> Thanks.Hope to read the user guide earlier.

Are you able to run the project? If yes, can you tell the procedure.
